File: cifrado/web/codigo/Ruby-Inesita/CifradosWeb/app/gestor.rb
require 'cesar.rb'
require 'reverse.rb'
require 'reverseG.rb'

class Gestor
    include Inesita::Injection

    attr_accessor :palabra
    attr_accessor :cesar
    attr_accessor :reverse
    attr_accessor :reverse_g

  
    def initialize
      @palabra = "gest"
      @cesar = Cesar.new
      @reverse = Reverse.new
      @reverse_g = ReverseG.new
    end

    def cifrar( pala )
        @cesar.palabra = pala
        @reverse.palabra = pala
        @reverse_g.palabra = pala

        @cesar.cifrar()
        @reverse.cifrar()
        @reverse_g.cifrar()

    end

    def cifradoSinActualizacion
        @cesar.cifrar()
        @reverse.cifrar()
        @reverse_g.cifrar()
    end
end

File: cifrado/web/codigo/Kotlin-Vaadin/ProyectoTestVaadin/vok-helloworld-app/web/src/main/kotlin/com/example/vok/gestor.kt
package com.example.vok

class Gestor
{
    var palabra = ""
    var cesar = Cesar()
    var reverse = Reverse()
    var reverse_g = ReverseG()

    fun cifrar(palab:String)
    {
        cesar.palabra = palab
        reverse.palabra = palab
        reverse_g.palabra = palab

        cesar.cifrar()
        reverse.cifrar()
        reverse_g.cifrar()
    }

    fun cifradoSinActualizacion()
    {
        cesar.cifrar()
        reverse.cifrar()
        reverse_g.cifrar()
    }
    
}